### PR TITLE
release: new version 2.5

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

release version 2.5

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by iamayushdas — APPROVED** [review]

**Review by imjoey — APPROVED** [review]
/lgtm

Great to see v2.5 is sufficiently close to being released. 😄  Thanks.

**Review by gxthrj — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +29,37 @@
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+# 2.5.0
+
+This release mainly improves basic features, bugfix and adds test cases.
+
+`Manager API` 2.5 should be used with [Apache APISIX 2.4](https://github.com/apache/apisix/releases/tag/2.4). It is not recommended to use with other Apache APISIX versions.
```
Comment (juzhiyuan) [inline CHANGELOG.md:36]: Would better use `http://apisix.apache.org/downloads` or `https://www.apache.org/dyn/closer.cgi/apisix/2.4/apache-apisix-2.4-src.tgz`  instead

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]
LGTM

**Comment by nic-chen** [comment]
Convert to draft, avoid clicking the merge button by mistake

**Review by starsz — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@nic-chen @Yiyiyimu  Should we keep this branch?

**Comment by bisakhmondal** [comment]
Nice🔥
I think we missed updating the readme.md
> The latest released version is 2.4 and should be used with Apache APISIX 2.3. It is not recommended to use with other Apache APISIX versions.

**Comment by moonming** [comment]
> Nice🔥
> I think we missed updating the readme.md
> 
> > The latest released version is 2.4 and should be used with Apache APISIX 2.3. It is not recommended to use with other Apache APISIX versions.

good catch. @nic-chen please create a new PR to fix.

**Comment by juzhiyuan** [comment]
> Nice🔥
> I think we missed updating the readme.md
> 
> > The latest released version is 2.4 and should be used with Apache APISIX 2.3. It is not recommended to use with other Apache APISIX versions.

oh!! yes, @bisakhmondal would like to update the README file & update `Release Guide` here? tells users what files need to update?

http://apisix.apache.org/docs/general/release-guide

**Comment by bisakhmondal** [comment]
> oh!! yes, @bisakhmondal would like to update the README file & update `Release Guide` here? tells users what files need to update?
> 
> http://apisix.apache.org/docs/general/release-guide

Yes, definitely!
Just confirming, do you mean updating the flow-chart which describes the required steps?

**Comment by nic-chen** [comment]
@bisakhmondal hi please help update readme first, thanks

**Comment by juzhiyuan** [comment]
> Just confirming, do you mean updating the flow-chart which describes the required steps?

More like a todo list? like:

- Update README in the master branch;
- Update README in the release branch;
- Update ManagerAPI Version in the release branch;
- Add Events on the apisix-website about releasing which project;

**Pull request merged**